### DISCUSSION TITLE
[Project Beta]: Support logical operators in filters

### DISCUSSION THREAD

**Discussion by danielhoherd** [community/community#15655] [Projects and Issues] [answered]
I'd love the ability to use logical operators in filters. Currently, filtering by `status:"In Progress" status:Closed ` comes back empty because it appears that these filters have an implicit AND between them. I'd love to be able to have an `OR` in there so I can filter by similar statuses.

**Comment by jwerre** [comment]
I have a similar issue (https://github.com/github/feedback/discussions/16044). That said, I was able to get the `-` operator to work so you could try something like this: 

```status:"In Progress" -status:Closed```

Translation: "Status is in progress AND NOT status is closed"

**Reply by danielhoherd** [reply]
`status` can only have one value though, so any `status:"In Progress"` inherently has `NOT status:Closed`. My original goal was to have an `OR` in there. Updated the description to indicate this.

**Comment by michael-hawker** [comment]
Yeah, I have a case where I want to aggregate external bugs in a view as well, but also our internal bugs tracking some other issues related to those technologies. This means I need to look for different labels when it's in our repository but not when it's elsewhere. i.e.

```
(label:external label:technology),repo:external/reponame
```

Building on the `,` as OR syntax available now, just adding parathesis could help a lot here as this would read as:

"Contains Label `external` and `technology`, OR it comes from the external/reponame repo."

This is also discussed in #16456

**Reply by danielhoherd** [reply]
FWIW in the project beta board's "Filter" field I cannot get the "`,` as OR syntax" that you mentioned to work.

**Reply by michael-hawker** [reply]
Yeah, I opened an issue about that too, the textbox is very unwieldy: https://github.com/orgs/github-community/discussions/18761

I think you need to put a space before typing ',' to get it back out of 'this is the text mode' and 'this is a token mode'...

**Comment by JonB451** [comment, marked as answer]
@danielhoherd  Does the following work for you? 

` status:"Closed","In Progress"`

**Reply by danielhoherd** [reply]
Thank you @JonB451!

**Comment by balat** [comment]
How can I, for example, filter all issues with no target-date or target-date>="2025-01-01"?
I used to do `-target-date:"<2025-01-01",no:target-date` but it does not work anymore.

**Reply by Waterstraal** [reply]
I am also looking for this... Is this not at all possible?
